cppsrc/U8x8lib.cpp

```diff
@@ -67,6 +67,7 @@ size_t U8X8::write(uint8_t v)
 /*=============================================*/
 /* callbacks */
 
+#ifdef U8X8_USE_PINS
```

Comment:
What exactly is the purpose of this #ifdef statement?
Did you test the lib with and without U8X8_USE_PINS define?

Reply:
The `u8x8_gpio_and_delay_arduino` function, which is guarded by this `#ifdef` uses other features (such as https://github.com/matthijskooijman/u8g2/blob/4f926cba3722017f7447a7a76649adb8dd4956b5/csrc/u8x8.h#L312 and https://github.com/matthijskooijman/u8g2/blob/4f926cba3722017f7447a7a76649adb8dd4956b5/csrc/u8x8.h#L343) which are already guarded by `U8X8_USE_PINS`. So without this added guard, when compiling without `U8X8_USE_PINS`, compilation would fail because these things were missing.

I tested this on a regular machine without `U8X8_USE_PINS`. I think I did not test on an Arduino with `U8X8_USE_PINS` defined, but in theory nothing should change when this define is present.

Reply:
yes, true